firstvoices/backend/tests/test_apis/test_characters_api.py

```diff
@@ -142,3 +143,176 @@ def test_detail_related_entries(self):
                 "type": entry1.type,
             }
         ]
+
+    @pytest.mark.django_db
+    def test_update_related_media(self):
```

Comment:
naive Q: would it be worthwhile to have a test where only 1 related media was updated, instead of all of them? (in light of the required=false change you made above) Or is this already covered by other tests?

Reply:
Good point! I'll split this test into 3